pynicotine/core.py

```diff
@@ -464,8 +464,8 @@ def _check_privileges(self, msg):
         days = hours // 24
 
         if msg.seconds == 0:
-            log.add(_("You have no Soulseek privileges. Privileges are not required, but allow your downloads "
-                      "to be queued ahead of non-privileged users."))
+            log.add(_("You have no Soulseek privileges. With privileges your downloads "
+                      "will always be first in the download queue."))
```

Comment:
I want to formulate it positively, explaining the advantage, so they might consider contributing.

Reply:
Sourced from https://www.slsknet.org/userlogin.php?username=user ...

> _**Donations, Perks and Privileges**_
>
> As a thank you for your support, for a donation of $5 or more:
> 
> - Your username will become permanent, never to be recycled after any period of disuse.
> - If you're using SoulseekQt, the 'Support Soulseek!' button will be permanently replaced with a gold star badge.
> - You'll receive a minimum of one month of privileges. While your privileges are active, your downloads will be queued ahead of those of non-privileged users. Days of privileges can also be gifted to other users._

In this case it only seems right to plagarise some copy from the official website, because no doubt there are legal matters concerning the language used when asking for payments from users.

> I want to formulate it positively, explaining the advantage,

I agree that is a good idea, but we can't over-promise anything.

Reply:
I must admit, "While your privileges are active, your downloads will be queued ahead of those of non-privileged users" is an apt formulation.